services/movie_session.py

```diff
@@ -41,4 +42,9 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    get_object_or_404(MovieSession, id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
```

Comment:
Consider specifying a more precise return type for `get_taken_seats`, such as `list[dict]`, to improve code clarity.